tests/core/test_core.py

```diff
@@ -266,18 +266,25 @@ def test_narrow_to_all_mentions(self, mocker, controller, index_all_mentions):
         msg_ids = {widget.original_widget.message["id"] for widget in widgets}
         assert msg_ids == id_list
 
-    def test_view_in_browser(self, mocker, controller):
+    @pytest.mark.parametrize(
+        "server_url",
+        [
+            "https://chat.zulip.org/",
+            "https://foo.zulipchat.com/",
+        ],
+    )
+    def test_view_in_browser(self, mocker, controller, message_fixture, server_url):
         # Set DISPLAY environ to be able to run test in CI
         os.environ["DISPLAY"] = ":0"
-        mock_get = mocker.patch("webbrowser.get", mocker.Mock())
-        mock_open = mocker.patch("webbrowser.open", mocker.Mock())
-        message_id = 123456
-        controller.model.server_url = "https://foo.zulipchat.com/"
+        controller.model.server_url = server_url
+        message_id = message_fixture["id"]
+        mocked_get = mocker.patch(CORE + ".webbrowser.get")
+        mocked_open = mocker.patch(CORE + ".webbrowser.open")
```

Comment:
These lines look better than the original, but there's no reason to change the names and move them around - the commit diff is bigger and less clear because of it.